hotel-service/src/main/resources/static/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
```

Comment:
Please exclude this file from the commit: it is not related to the current task.